benchcmp.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash          
+
+# usage: ./benchcmp.sh <commit-sha1> <commit-sha2>
+# Exit on any non zero return value on execution of a command. 
+set -e 
+
+# path of benchcmp. 
+benchcmp=${GOPATH}/bin/benchcmp
+
+# function which runs the benchmark comparison. 
+RunBenchCmp () {
+	# Path for storing output of benchmark at commit 1. 
+	commit1Bench=/tmp/minio-$1.bench
+	# Path for storing output of benchmark at commit 2. 
+	commit2Bench=/tmp/minio-$2.bench
+	# switch to commit $1. 
+	git checkout $1
+	# Check if the benchmark results for given commit 1 already exists. 
+	# Benchmarks are time/resource consuming opertions, run only if the the results doesn't exist. 
+	if [[ ! -f $commit1Bench  ]]
+	then
+
+		echo "Running benchmarks at $1"
+		go test -bench=. | tee $commit1Bench 
+	fi
+	# get back to the commit from which it was started. 
+	git checkout -
+	echo "Checking into commit $2"
+	# switch to commit $2 
+	git checkout $2
+	# Check if the benchmark results for given commit 2 already exists. 
+	# Benchmarks are time/resource consuming opertions, run only if the the results doesn't exist. 
```

Comment:
'opertions' --> 'operations' ,  'run only if the results doesn't exist.'  remove an extra 'the'